wdl/transforms/new-base/src/main/scala/wdl/transforms/base/linking/expression/types/EngineFunctionEvaluators.scala

```diff
@@ -139,7 +139,10 @@ object EngineFunctionEvaluators {
   implicit val writeJsonFunctionEvaluator: TypeEvaluator[WriteJson] = new TypeEvaluator[WriteJson] {
     override def evaluateType(a: WriteJson, linkedValues: Map[UnlinkedConsumedValueHook, GeneratedValueHandle])
                              (implicit expressionTypeEvaluator: TypeEvaluator[ExpressionElement]): ErrorOr[WomType] = {
-      validateParamType(a.param, linkedValues, WomObjectType).map(_ => WomSingleFileType)
+      a.param.evaluateType(linkedValues).flatMap {
+        case v if WomArrayType(WomAnyType).isCoerceableFrom(v) | WomObjectType.isCoerceableFrom(v) => v.validNel
```

Comment:
Intentional bitwise OR? Should work fine but meaning unclear.
```suggestion
        case v if WomArrayType(WomAnyType).isCoerceableFrom(v) || WomObjectType.isCoerceableFrom(v) => v.validNel
```

Reply:
Sorry this was not intentional. I have updated it to `||`